app/models/like.rb

```diff
@@ -1,4 +1,9 @@
 class Like < ApplicationRecord
-  belongs_to :author_id
-  belongs_to :post_id
+  belongs_to :author, class_name: :User
+  belongs_to :post, class_name: :Post
+
+  def increase_like_counter
+    post.likes_counter += 1
+    post.save
+  end
```

Comment:
- **[OPTIONAL]** I think it would be nice if you triggered this method using the `after_save` active record callback so that when a like is saved Rails automatically calls this function. I also think we can refactor this code to be more brief.

  ```
  .
  .
  after_save :increase_like_counter
  .
  .
  private

  def increase_like_counter
    post.increment!(:likes_counter)
  end
  ```